site/static/examples/kind-gcr.sh

```diff
@@ -27,4 +37,4 @@ for node in $(kind get nodes --name "${KIND_CLUSTER_NAME}"); do
   docker exec "${node_name}" systemctl restart kubelet.service
 done
 
-echo "Done!"
\ No newline at end of file
+echo "Done!"
```

Comment:
this renders a spurious blank line in the docs which we haven't had time to fix

Reply:
Thanks!